main-actions/src/main/scala/sbt/Sync.scala

```diff
@@ -35,19 +37,22 @@ object Sync {
       store: CacheStore,
       inStyle: FileInfo.Style = FileInfo.lastModified,
       outStyle: FileInfo.Style = FileInfo.exists,
+      fileConverter: FileConverter
   ): Traversable[(File, File)] => Relation[File, File] =
-    sync(store, inStyle)
+    sync(store, inStyle, fileConverter)
```

Comment:
I guess since `apply(...)` is already deprecated we don't have to add `fileConverter` here once we have backward compatible `sync(...)`.